javascript/builtins/TypedArray.json

```
@@ -344,10 +344,10 @@
                 "version_added": false
               },
               "safari": {
-                "version_added": false
+                "version_added": "10"
               },
               "safari_ios": {
-                "version_added": false
+                "version_added": "10"
```

```suggestion
                "version_added": "9.3"
```
…per https://trac.webkit.org/log/webkit/tags/Safari-601.2.7.1 and the following:

https://github.com/mdn/browser-compat-data/blob/17a48dcd70cc8c804ffc53d904441417a01263d5/browsers/safari_ios.json#L96-L100